src/convertToObject.js

```diff
@@ -5,8 +5,25 @@
  *
  * @return {object}
  */
-function convertToObject(sourceString) {
-  // write your code here
-}
+
+const convertToObject = (stylesString) => {
+  const styleObject = {};
```

Comment:
[CODE STYLE] - The variable name 'styleObject' could be more descriptive. Consider renaming it to reflect that it represents the styles parsed from the string, such as 'parsedStyles' or 'styleMap'.